crates/database/src/orders.rs

```diff
@@ -704,6 +666,41 @@ pub async fn update_limit_order_fees(
         .await
 }
 
+/// All data required to filter, select and update orders to update the
+/// `surplus_fee` for.
+#[derive(Debug, Clone, sqlx::FromRow, PartialEq, Eq, Default)]
+pub struct OrderQuotingData {
+    pub uid: OrderUid,
+    pub owner: Address,
+    pub sell_token: Address,
+    pub buy_token: Address,
+    pub sell_amount: BigDecimal,
+    pub sell_token_balance: SellTokenSource,
+    pub partially_fillable: bool,
+}
+
+/// Returns all limit orders that are currently waiting to be filled.
+pub fn open_limit_orders(
```

Comment:
Can we add a comment here indicating that they are sorted by surplus_fee_timestamp (so the most outdated orders come first)?